examples/digital_in/wfm_acq_int_clk.py

```diff
@@ -0,0 +1,16 @@
+﻿"""Example for reading digital signal.
+
+This example demonstrates how to input a finite digital pattern
+using the DAQ device's internal clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, LineGrouping, READ_ALL_AVAILABLE
+
+
+with nidaqmx.Task() as task:
+    task.di_channels.add_di_chan("Dev1/port0", line_grouping=LineGrouping.CHAN_FOR_ALL_LINES)
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.FINITE, samps_per_chan=50)
+
+    data = task.read(READ_ALL_AVAILABLE)
+    print("Acquired data: [" + ", ".join(f"{value:f}" for value in data) + "]")
```

Comment:
Digital values are not floats. With a simulated PXIe-6363, this prints

> Acquired data: [0.000000, 16843009.000000, 33686018.000000, 50529027.000000, 67372036.000000, 84215045.000000, 101058054.000000, 117901063.000000, 134744072.000000, 151587081.000000, 168430090.000000, 185273099.000000, 202116108.000000, 218959117.000000, 235802126.000000, 252645135.000000, 269488144.000000, 286331153.000000, 303174162.000000, 320017171.000000, 336860180.000000, 353703189.000000, 370546198.000000, 387389207.000000, 404232216.000000, 421075225.000000, 437918234.000000, 454761243.000000, 471604252.000000, 488447261.000000, 505290270.000000, 522133279.000000, 538976288.000000, 555819297.000000, 572662306.000000, 589505315.000000, 606348324.000000, 623191333.000000, 640034342.000000, 656877351.000000, 673720360.000000, 690563369.000000, 707406378.000000, 724249387.000000, 741092396.000000, 757935405.000000, 774778414.000000, 791621423.000000, 808464432.000000, 825307441.000000]

```suggestion
    print("Acquired data: [" + ", ".join(f"{value:#x}" for value in data) + "]")
```
Updated output:
> Acquired data: [0x0, 0x1010101, 0x2020202, 0x3030303, 0x4040404, 0x5050505, 0x6060606, 0x7070707, 0x8080808, 0x9090909, 0xa0a0a0a, 0xb0b0b0b, 0xc0c0c0c, 0xd0d0d0d, 0xe0e0e0e, 0xf0f0f0f, 0x10101010, 0x11111111, 0x12121212, 0x13131313, 0x14141414, 0x15151515, 0x16161616, 0x17171717, 0x18181818, 0x19191919, 0x1a1a1a1a, 0x1b1b1b1b, 0x1c1c1c1c, 0x1d1d1d1d, 0x1e1e1e1e, 0x1f1f1f1f, 0x20202020, 0x21212121, 0x22222222, 0x23232323, 0x24242424, 0x25252525, 0x26262626, 0x27272727, 0x28282828, 0x29292929, 0x2a2a2a2a, 0x2b2b2b2b, 0x2c2c2c2c, 0x2d2d2d2d, 0x2e2e2e2e, 0x2f2f2f2f, 0x30303030, 0x31313131]

Reply:
My mistake, the printout is corrected to hexadecimal values.